### PR TITLE
Make nodes in multi validator genesis have different monikers

### LINKED ISSUE #242: Make nodes in multi-validator genesis have different monikers
This is so that in testnet we clearly see different behavior of different validators on `scan`.

### PR DESCRIPTION
**Pull request by prin-r**
Fixes #242

### CODE REVIEW

**Comment by sorawit** [comment]
How did you test this change?

**Review by taobun — APPROVED** [review]

File: chain/docker-config/generate-genesis.sh [hunk 1]
```diff
@@ -63,20 +69,29 @@ echo "12345678" | bandd gentx \
...
 echo "12345678" | bandd gentx \
     --amount 100000000uband \
     --node-id 7b58b086dd915a79836eb8bfa956aeb9488d13b0 \
     --pubkey bandvalconspub1addwnpepqwj5l74gfj8j77v8st0gh932s3uyu2yys7n50qf6pptjgwnqu2arxkkn82m \
     --name validator3 \
     --ip 172.18.0.13
 
+# modify moniker
+sed 's/node-validator/node-validator-4/g' ~/.bandd/config/config.toml.temp > ~/.bandd/config/config.toml
+
```
Comment (taobun) [inline chain/docker-config/generate-genesis.sh:94]: Remove temp file after used

**Review by taobun — CHANGES_REQUESTED** [review]

**Review by sorawit — DISMISSED** [review]
Looks great!

**Review by evilpeach — APPROVED** [review]
LGTM

**Review by taobun — APPROVED** [review]
lgtm

**Review by sorawit — APPROVED** [review]
Looks good!

**Pull request merged**